add tips and tricks (#553)
wizardsardine/liana: doc/TRY.md

rm -rf ~/.bitcoin/signet
rm -rf ~/.liana/signet
```

## Tips & Tricks 

> If you want to start several instance of Liana (for simulate different participants on the same machine by example), 
> you can specify the datadir path for each:

Reviewer: ```suggestion
You can simulate multiple wallets by using different data directories. For instance:
```

```
./liana-gui --datadir test_alice
./liana-gui --datadir test_bob
./liana-gui --datadir test_charlie
 ```

> If the directory not exist, Liana will make it.

Reviewer: ```suggestion
The directory will be created if it doesn't exist.
```
